spec/units/scorecard_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'scorecard.rb'
+
+describe Scorecard do
```

Comment:
It might be helpful to increase the test coverage to include full games (e.g. perfect game), especially where the final frame has 3 rolls